.github/workflows/benchmark.yml

```diff
@@ -88,7 +88,7 @@ jobs:
 
       - name: "Install dependencies"
         if: ${{ github.event_name == 'pull_request' }}
-        run: source velox/scripts/setup-ubuntu.sh
+        run: source velox/scripts/setup-ubuntu.sh && install_build_prerequisites &&  install_velox_deps_from_apt
```

Comment:
argument `install` is ambiguous since it does not specify what to install.
I can add another function `install_apt_deps` which includes the functions `install_build_prerequisites`  and `install_velox_deps_from_apt`. How does that sound?
`source velox/scripts/setup-ubuntu.sh && install_apt_deps`

Reply:
Thanks Deepak - that would also work